spec/spec_helper.rb

```diff
@@ -84,8 +84,11 @@ def sexp(type, *children)
   end
 
   # Avoid infinitely running tests. This is mainly useful when running mutant.
-  config.around(:each) do |example|
-    Timeout.timeout(5, &example)
+  # Set DEBUG to disable this and allow using pry without specs failing.
```

Comment:
Nope, you just need to set it. As in, 'to' here is *not* part of a "Set DEBUG to '1' to disable this". Maybe it's too ambiguous :smile:.